docs/reference/openapi-rules.md

```diff
@@ -253,10 +253,6 @@ Why? Well, you _can_ reference tags arbitrarily in operations, and definition is
 
 Defining tags allows you to add more information like a `description`. For more information see [tag-description](#tag-description).
 
-**Recommended:** Yes
-
-### operation-default-response
```

Comment:
This makes very little sense to me, as the spec only says MAY have a default response.